app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,11 @@
         <activity
             android:name=".activities.SplashScreen"
             android:theme="@style/AppTheme.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
```

Comment:
because we need to the fetch Susi Base URLs list every time whenever app will open, we cannot wait to any SocketTimeoutException come and this is the best place to fetch.
we can not wait for SocketTimeoutException because if any SocketTimeoutException came during fetching susi response we cannot make new request to susi server to ask which server is running, this current is not.